ezyrb/neighborsregressor.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from .approximation import Approximation
+
+
+class NeighborsRegressor(Approximation):
+    """
+    A generic superclass for wrappers of *NeighborsRegressor from sklearn.
+
+    :param kwargs: arguments passed to the internal instance of
+        *NeighborsRegressor.
+    """
+
+    def __init__(self, **kwargs):
+        raise ValueError("This class must be extended, not used.")
```

Comment:
Please check that this is the correct error: https://docs.python.org/3/library/exceptions.html